src/Exception/TooManyRequestsHttpException.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Akeneo\Pim\ApiClient\Exception;
+
+/**
+ * @copyright 2022 Akeneo SAS (https://www.akeneo.com)
+ * @license   https://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class TooManyRequestsHttpException extends ClientErrorHttpException
+{
+    /**
+     * How much time must be waited before next request. Result in seconds.
+     */
+    public function getRetryAfter(): int
+    {
+        $retryAfterHeader = $this->getResponse()->getHeader('Retry-After');
+
+        return (int) $retryAfterHeader[0];
+    }
+
+    /**
+     * How much time must be waited before next request. Result in milliseconds.
+     */
+    public function getRetryIn(): int
+    {
+        $retryInHeader = $this->getResponse()->getHeader('X-Retry-In');
+
+        return ceil((float) $retryInHeader[0]);
```

Comment:
failing/notice if the header does not exist

Reply:
Method removed